plugins.json

```diff
@@ -3657,6 +3657,177 @@
     },
     "updated_at": "2019-06-11T19:41:07Z"
   },
+  {
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+      "icon_data": "",
+      "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-calls-v0.12.0.tar.gz",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.12.0",
+      "hosting": "",
+      "author_type": "mattermost",
+      "release_stage": "production",
+      "enterprise": false,
+      "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmO8WFsACgkQ0bVLR6XO/sQk1g/8DvH3cg1TqNfQ1eigJyqreSqSBpNBjj1hzMtaRuELs0gMOvg/VlT1MDO3A4NKmsFsIggx0aAuSAjnOZCihp8cfIvm3e5YJmaUBJdLRbSDb4usyNlz35DyB8xjytWeZLDKDNO5xNvEj4L6QbtJIwxFczY9/NDcpd0KbqK6Fx+ws6D1l22kTHvE6y9vlc5AZi0LVvyEl60QZnHvddDJmc//vKVc7a7ej6snQ9IAwAlMttDRQ/g7Wbs3LVyWGoP7vTXRM8BbGF0VCfCUXMLKjp9mFK16iPmEp3o09iBcnvk4CeAzjZqaf1At1BO5PC9g8KaBCdjBHSK2IeTPFSj8ZViwqQl8PDK0iHO1SGFTknUw1g+mrG++noiyxaeE45iMd8ZKTJqhVZuMbWdXI+mILxZUFSwjHjxX3rf6G4/Papk/WV6+C95JoP9N2upNyM7wq0ziBqV6SOOYpH3FJqvxIiXOpDn1Qlr6LcPIvV0pH1J5HXdbmi0F5nWYrGJ7q8dpuaHZs8U36b/VqTS77VFPxML2OmYUDst0t5bZIP4KLYAJ8VqH3+OaLs3K368TgY/k8gG4IYLI6hpSJsSI5a2Z4qcN1qMw9DCbYYihj179LZZYhb922tuw84n4LIgeutR0PV29m6/opx4Wpyl+x856Hi8rfi3fCEHdNxnvv2gShPZ8wRc=",
+      "repo_name": "mattermost-plugin-calls",
+      "manifest": {
+          "id": "com.mattermost.calls",
+          "name": "Calls",
+          "description": "Integrates real-time voice communication in Mattermost",
+          "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+          "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+          "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.12.0",
+          "version": "0.12.0",
+          "min_server_version": "7.6.0",
```

Comment:
nit: given 7.6 doesn't really exist, should this be 7.7?

Reply:
That's a good point. I am not opposed to change it here but to have it in the plugin we'd have to go back and restart the release process. Not sure if worth it but happy to hear thoughts. @cpoile?

Reply:
I don't mind the extra work if it makes it cleaner. But on the other hand, nothing will break, so 🤷 . Up to you @streamer45

Reply:
Let's change it here then and will fix it on the plugin side starting in next release.